core/src/main/java/org/apache/calcite/adapter/jdbc/JdbcTable.java

```diff
@@ -130,7 +130,7 @@
 
   private List<Pair<ColumnMetaData.Rep, Integer>> fieldClasses(
       final JavaTypeFactory typeFactory) {
-    final RelDataType rowType = requireNonNull(protoRowType, "protoRowType").apply(typeFactory);
+    final RelDataType rowType = getRowType(typeFactory);
```

Comment:
Just for the reference, the previous code was `protoRowType.apply(typeFactory)` (see https://github.com/apache/calcite/blob/2f2fc406f4675a9ca16e3c0a352cfa4f2a1bc047/core/src/main/java/org/apache/calcite/adapter/jdbc/JdbcTable.java#L130), so nullness verifier was right that `protoRowType` should not be used like that.

I added `requireNonNull` just to avoid unrelated changes.